packages/ppx_monadic/ppx_monadic.1.0.4/descr

```diff
@@ -0,0 +1,10 @@
+ppx_monadic: ppx extension for do notation, pattern guards, and monad comprehension 
```

Comment:
could you remove the `ppx_monadic:` from the description on the first line? it'll be repeated in the UI otherwise